app/models/schemas.py

```diff
@@ -76,12 +81,13 @@ class RatingCreate(RatingBase):
 # Download Schemas
 
 class DownloadBase(BaseModel):
-    ip_address: str
+    user: str
```

Comment:
Meaning we should be writing userId to the Download table right?
Maybe I'm confused but I can't see where the user is used to create a download entity on the router. Same for referral

Reply:
just push my last change, it should be user_id not user